Content.Server/_DV/Abilities/Kitsune/KitsuneFoxSystem.cs

```diff
@@ -0,0 +1,31 @@
+using Content.Server.Polymorph.Components;
+using Content.Server.Polymorph.Systems;
+using Content.Shared._DV.Abilities.Kitsune;
+using Content.Shared.Damage.Systems;
+using Content.Shared.StatusEffect;
+using Content.Shared.Stunnable;
```

Comment:
Unused using directive
```suggestion
using Content.Shared.Damage.Systems;
using Content.Shared.Stunnable;
```